google/cloud/storage/blob.py

```diff
@@ -1628,7 +1628,9 @@ def download_as_text(
             return data.decode(encoding)
 
         if self.content_type is not None:
-            _, params = cgi.parse_header(self.content_type)
+            message = Message()
```

Comment:
you can also use HeaderParser https://github.com/aio-libs/aiohttp/pull/6708/files

Reply:
Thanks, that is clearer.